src/client/profile-provider.ts

```diff
@@ -67,11 +67,12 @@ function profileAstId(ast: ProfileDocumentNode): string {
 const boundProfileProviderDebug = createDebug(
   'superface:bound-profile-provider'
 );
+export type AuthCache = { cache?: { digest?: string } };
```

Comment:
shouldn't there be some sort of cache key?

Reply:
This exists on boundProfileProvider so it is already scoped only on one provider. Not sure if something more is needed:
 - if user reuses boundProfileProvider with different auth. method this won't be used